document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/handler/CompanyReportDataHandler.java

```diff
@@ -74,6 +79,15 @@ private String getCompanyReportData(String companyNumber,  String requestId) thr
         CompanyProfileApi companyProfileApi = getCompanyProfile(companyNumber, requestId);
         companyReportApiData.setCompanyProfileApi(companyProfileApi);
 
+        if (companyProfileApi.getLinks().containsKey(PSCS_KEY)) {
+            try {
+                PscsApi pscsApi = getPscs(companyNumber, requestId);
+                companyReportApiData.setPscsApi(pscsApi);
```

Comment:
can just be `companyReportApiData.setPscsApi(getPscs(companyNumber, requestId));`